### PR TITLE
Fixed trial stage progress bug (upon clicking/arrow down) + added meaningful constants for certain trial stages

### PR DESCRIPTION
**Pull request by nija-computas**

### CODE REVIEW

File: src/components/tetris/Tetris.tsx [hunk 1]
```diff
@@ -329,7 +340,8 @@ export default function Tetris() {
       stop();
     }
 
-    if (state.trialStage === 3) {
+    if (state.trialStage == TRIAL_PLAY) {
```
Comment (och-compuas) [inline src/components/tetris/Tetris.tsx:343]: Consider using triple = for type safe comparison (===)

File: src/components/tetris/Tetris.tsx [hunk 2]
```diff
@@ -247,7 +254,11 @@ export default function Tetris() {
   };
 
   const moveMaxDown = (): void => {
-    if (state.gameOver || state.startScreen) {
+    if (
+      state.gameOver ||
+      state.startScreen ||
+      (state.trial && state.trialStage != TRIAL_PLAY)
```
Comment (och-compuas) [inline src/components/tetris/Tetris.tsx:260]: Consider using type safe comparison ( !== )

File: src/components/tetris/Tetris.tsx [hunk 3]
```diff
@@ -257,11 +268,11 @@ export default function Tetris() {
...
   const drop = (): void => {
-    if (state.gameOver || state.startScreen) {
-      return;
-    }
-
-    if (state.trial && state.trialStage !== 3) {
+    if (
+      state.gameOver ||
+      state.startScreen ||
+      (state.trial && state.trialStage != TRIAL_PLAY)
```
Comment (och-compuas) [inline src/components/tetris/Tetris.tsx:274]: Consider using type safe comparison ( !== )

File: src/components/trial/TrialScreen.tsx [hunk 4]
```diff
@@ -39,6 +42,10 @@ const TrialScreen = (props: TrialScreenProps) => {
     }
   };
 
+  const progressClick = () => {
+    if (trialStage == 2) progressTrial();
```
Comment (och-compuas) [inline src/components/trial/TrialScreen.tsx:46]: Consider using type safe comparison ( === )
Also consider putting the code inside brackets { }

**Review by och-compuas — CHANGES_REQUESTED** [review]
Just some minor changes needed

**Comment by nija-computas** [comment]
Fixed requested changes.

**Review by och-compuas — APPROVED** [review]

**Pull request merged**